README.rst

```diff
@@ -127,3 +127,34 @@ Some notes about translating the guide:
 
 To test your translation, use ``make BUILDER-LANGUAGE`` command (for example,
 ``make html-it`` will build HTML docs in Italian language).
+
+Pre-commit checks
+-----------------
+
+Optionally, you can run pre-commit checks on your changes. To do so:
+
+1. Install the ``pre-commit`` package from the Ubuntu repositories::
+
+    sudo apt install pre-commit
+
+2. Install the ``woke`` snap by running::
+
+    sudo snap install woke
+
+   Alternatively, you can install it by following instructions in `<https://docs.getwoke.tech/installation/>`_.
```

Comment:
Question: Do we want to add the installation of `woke` and `pre-commit` to the Makefile?

Reply:
@dviererbe I have added this after refactoring the Makefile to avoid duplication.